pkg/controller/infra/infra_controller.go

```diff
@@ -100,6 +100,7 @@ func NewInfraController(clusterInformer informers.ClusterInformer, kubeClient ku
 
 	c.syncHandler = c.syncCluster
 	c.enqueueCluster = c.enqueue
+	c.ansibleRunner = ansible.NewAnsibleRunner(kubeClient)
```

Comment:
Would you have any qualms about the ansible runner being injected as a parameter to `NewInfraController`? I thought about making that change for #28, but ultimately decided not to. With the changes here, I am leaning again towards providing the ansible runner as a parameter. Note that I am not suggesting making any change for this PR here; I only wanted to know your thoughts on it as a possibility of making the change in my PR.

Reply:
Sounds good to me.